modules/ocf_desktop/manifests/packages.pp

```diff
@@ -21,6 +22,8 @@
       'gnome-icon-theme', 'redshift', 'xfce4-whiskermenu-plugin']:;
     # desktop helpers
     ['libimage-exiftool-perl']:;
+    # development
+    ['docker-compose']:;
```

Comment:
is this really new enough for you? personally i'd just install this in a venv instead

Reply:
Also this package isn't desktop specific, so if we wanted to install this it would be better to define this resource in `ocf::packages::docker` and make it optionally installed depending on the value of a parameter.

Reply:
@chriskuehl actually youre right, its way too old. ill remove this commit